addons/crm_iap_lead_enrich/models/crm_iap_lead_enrich.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import datetime
+
+from odoo import fields, models, api, _
+from odoo.exceptions import UserError
+from odoo.addons.iap import jsonrpc,InsufficientCreditError
+
+DEFAULT_ENDPOINT = 'https://iap-services.odoo.com'
+
+class Lead(models.Model):
+    _inherit = 'crm.lead'
+
+    enriched_lead = fields.Boolean(string='Lead enriched based on email', default=False)
```

Comment:
As False is already the defualt value of a boolean field there is no need to force it.